awx_collection/plugins/doc_fragments/auth_plugin.py

```diff
@@ -18,42 +18,42 @@ class ModuleDocFragment(object):
     env:
     - name: CONTROLLER_HOST
     - name: TOWER_HOST
-    deprecated:
-    - removed_from_collection: 'awx.awx.controller'
-    - removed_in: '4.0.0'
-    - why: Collection name change
-    - alternative: 'CONTROLLER_HOST'
+      deprecated:
+        removed_from_collection: 'awx.awx.controller'
```

Comment:
This is **not** a collection name. The collection is `awx.awx`:
```suggestion
        removed_from_collection: 'awx.awx'
```
Same in the others.

Not sure whether it should be fixed in this PR though, or in a follow-up.